### PR TITLE
Updated for psc v0.10.1

### PR DESCRIPTION
**Pull request by jacereda**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -11,9 +11,22 @@
...
-    "purescript-math": "~0.1.1"
+    "purescript-arrays": "^3.0.0",
+    "purescript-foldable-traversable": "^2.0.0",
+    "purescript-integers": "^2.0.0",
+    "purescript-canvas": "^1.0.0",
+    "purescript-lists": "^3.0.0",
+    "purescript-math": "^2.0.0"
+  },
+  "resolutions": {
+    "purescript-foldable-traversable": "^2.0.0",
```
Comment (paf31) [inline bower.json]: Why does this need resolutions? Everything above should be compatible.

File: src/Main.purs [hunk 2]
```diff
@@ -121,25 +119,21 @@ scene t = do
...
 main = do
-  Just canvas <- getCanvasElementById "canvas"
+  mcanvas <- getCanvasElementById "canvas"
+  let canvas = fromMaybe' (\_ -> unsafeCrashWith "no canvas element") mcanvas
+
   ctx <- getContext2D canvas
 
-  fortyFps \t -> do
-    clearRect ctx { x: 0, y: 0, w: 1000, h: 1000 }
+  atFps 30.0 \t -> do
```
Comment (paf31) [inline src/Main.purs:131]: Any reason for the different frame rate?

**Comment by paf31** [comment]
Looks good, thanks. Could you please merge with `master`, I've added the missing LICENSE file now. Thanks.

File: src/Main.purs (hunk 2 above)
Comment (jacereda) [reply src/Main.purs:131]: Just that 30 or 60 FPS looks better than 40 in normal 60Hz monitor frequencies.

**Comment by paf31** [comment]
Great, thanks very much!

**Pull request merged**